(Web-Embeds) Remove unsupported types for all routing switches

linb1 opened the pull request:
## 🐛 Issue

[(Web-Embeds) Remove unsupported types for all routing switches](https://3.basecamp.com/3926363/buckets/27088350/card_tables/cards/6830930260/edit?focus=kanban_card_title)

There are unsupported content item types that determine routing that need to be removed

## ✏️ Solution

Remove all  unsupported content item types

## 🔬 To Test

1. Check route switching content item files like `getURLFromType`
2. It should only have these content item types: MediaContentItem, WeekendContentItem, UniversalContentItem, ContentSeriesContentItem
3. Run embeds and make sure everything is working the same
4. Unsupported types will probably no longer work

jordanwade commented on packages/web-shared/utils/getContentFromURL.js:
```diff
@@ -19,58 +15,35 @@ function getContentFromURL(url) {
 
   switch (type) {
-    case 'EventContentItem':
+    // case 'EventContentItem':
```
@linb1 you can just delete this instead of commenting it out.

linb1 replied:
@jordanwade whoops, forgot to delete that. I also deleted a couple other instances of it 👍

jordanwade reviewed (changes requested):
@linb1 I think we are doing this switch on type stuff in a couple of places, can you double check the codebase and make sure to update the other ones. One is `getURLFromType` the other ones might not be used in the app and we should probably tag those to be removed if we aren't using them anywhere in the app. We can do that in another PR, but definitely check out the `getURLFromType`. I think that one does need to be updated.

linb1 commented:
@jordanwade I think I removed the unsupported cases in `getURLFromType`, unless I'm misunderstanding what I need to do 😅. I think I looked through the codebase searching for where the unsupported content types might be. I also added a comment on if `InformationalContentSingle.js` should be removed since we wouldn't support `InformationalContentItem` anymore. These were all the ones I went after in this PR:

<img width="320" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/68402088/69467330-a907-47cd-b3cd-38178578dd9a">
